### PR TITLE
Blog app: add devise

### PR DESCRIPTION
**Pull request by Piercel2022**
The following requirements have been implemented:

-     Can register a new user.

-     User logs in with a combination of email and password.

-     Hashed password should be stored in the database.

-     Ask for confirmation of email.

-     Password can be reset
The following design has been implemented
blog_login

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
```
Comment (tufoinnkuo10) [inline app/models/user.rb:3]: - [ ] please let's kindly remove the commented code as it makes your code unclean and difficult to read

File: app/models/user.rb [hunk 2]
```diff
@@ -1,4 +1,8 @@
...
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```
Comment (tufoinnkuo10) [inline app/models/user.rb:5]: - [ ] nice job so far. please in other for the application to asks for confirmation of email ,please kindly ensure that `:confirmable` is enabled in the user model

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @TEAM

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Gitflow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally

Required Changes ♻️
Check the comments under the review

tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by Reem-lab — APPROVED** [review]
# Hi Team,👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Followed Gitflow ✅
- Great working on the functionality✅
- Professional README ✅
- Descriptive PR 💯💯


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**